bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/BusAux.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0" bindingId="openwebnet"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Thing for BUS Auxiliary commands -->
+	<thing-type id="bus_aux">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bus_gateway"/>
+		</supported-bridge-type-refs>
+
+		<label>Auxiliary</label>
+		<description>A OpenWebNet BUS/SCS Auxiliary command</description>
+
+		<channels>
+			<channel id="aux" typeId="aux"/>
+		</channels>
+
+		<properties>
+			<property name="vendor">BTicino/Legrand</property>
+			<!--<property name="model">BTI-F52x</property> -->
```

Comment:
Please remove this commented line if not needed

Reply:
Ok  the comment has been  removed